assets/css/services.scss

```diff
@@ -0,0 +1,69 @@
+$default-font: "Roboto", "Times New Roman", serif;
+$second-font: "Raleway", "Times New Roman", serif;
+
+$mobile: 375px;
+$tablet: 768px;
+$desktop: 1280px;
+$large-desktop: 1560px;
+
+$light-green: #497557;
+$green: #0c4a34;
+$grey: #757575;
+$dark-green: #02291c;
+$text-color-dark: #214e41;
+$text-color-light: #43806c;
+$white: #ffffff;
+
+.services__title {
+  text-align: center;
+  font-family: $default-font;
+  color: $text-color-dark;
+  margin: 12% auto;
+
+  @media (min-width: $tablet) {
+    margin: 4% auto;
+  }
+}
+
+.services-block__title,
+.services-block__text {
+  color: $text-color-dark;
+}
+
+.services {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  text-align: center;
+
+  @media (max-width: $mobile) and (max-width: $tablet) {
```

Comment:
@media (max-width: $mobile) and (max-width: $tablet) {
    flex-direction: column;
  }
  
  Extra media here